core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
I'm not sure why we'd do that, though.  If people aren't using analytics, we wouldn't want an extra JS include, and if they are, they can just download what google provides and use block override + block.parent to include it as needed.  At the least, I'd much prefer this in an optional app so it could have its own settings (for GA code) and be enabled or disabled by changing INSTALLED_APPS.